OrbitGl/Timer.h

```diff
@@ -19,12 +19,12 @@ class Timer {
 
   [[nodiscard]] uint64_t ElapsedNanos() const { return EndOrNow() - start_; }
   [[nodiscard]] double ElapsedMicros() const { return static_cast<double>(ElapsedNanos()) * 0.001; }
-  [[nodiscard]] double ElapsedMillis() const { return ElapsedMicros() * 0.0001; }
-  [[nodiscard]] double ElapsedSeconds() const { return ElapsedMillis() * 0.0001; }
+  [[nodiscard]] double ElapsedMillis() const { return ElapsedMicros() * 0.001; }
```

Comment:
maybe change it to "/ 1000.0"? It seems more readable that way.

Reply:
Sounds fair, done.